.github/workflows/pull-request-without-code.yml

```diff
@@ -24,13 +24,18 @@ jobs:
   build:
     strategy:
       matrix:
-        os: [ ubuntu-latest, windows-latest ]
-        # When changing the list of JDK versions, the build configuration has to be changed by a repository admin. See
-        # https://github.com/pinterest/ktlint/pull/1787#issuecomment-1409074092
-        jdk: [ 8, 11, 17, 19 ]
-        exclude: # windows with JDK8 are *really* flaky
-          - os: windows-latest
-            jdk: 8
+        os: [ubuntu-latest, windows-latest]
     runs-on: ${{ matrix.os }}
+    name: "[build] OS: ${{ matrix.os }}"
+    steps:
+      - run: 'echo "No build required"'
+
+  tests:
```

Comment:
This step should also be mentions in the comment on line 23.

Reply:
I noticed the comment was initially added to the whole `steps` block, so instead of adding a comment, I updated the existing one to mention all steps rather than just `jobs.build`. That's what I believe was the original intention (to keep all jobs in sync, regardless of their number 👀